packages/mux-player/package.json

```diff
@@ -96,7 +96,7 @@
   "dependencies": {
     "@mux/mux-video": "0.17.3",
     "@mux/playback-core": "0.22.2",
-    "media-chrome": "~2.2.5"
+    "media-chrome": "^3.0.2"
```

Comment:
stick to patch versions?

Reply:
ah yeah good catch/callout. 🦅 👀. Def agree.